pkg/trace/test/testsuite/traces_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"time"
```

Comment:
Can you please add an additional sub-test here which ensures that those root tags (`_dd.origin`, `_dd.hostname`, etc) correctly get promoted to the chunk fields?

Reply:
I added tests in `normalizer_test.go` and `agent_test.go`.